sdk/src/main/java/ch/admin/bag/covidcertificate/sdk/android/repository/CovidCertificateDatabaseRepository.kt

```diff
@@ -17,21 +16,20 @@ internal class CovidCertificateDatabaseRepository private constructor(context: C
 	NationalRulesStore {
 
 	companion object : SingletonHolder<CovidCertificateDatabaseRepository, Context>(::CovidCertificateDatabaseRepository) {
-		private const val ASSET_DB_PATH = "covid_certificate_database.db"
-
-		fun getPrepopulatedSinceHeader(environment: SdkEnvironment) = when (environment) {
-			SdkEnvironment.DEV, SdkEnvironment.ABN -> "0"
-			SdkEnvironment.PROD -> "11667563"
-		}
+		private const val ASSET_DB_PATH = "covid_certificate_database.sqlite"
 	}
 
 	private val database by lazy {
+		val path = context.getDatabasePath("revoked-certificates-db")
```

Comment:
I would extract this value into a constant, maybe named `LEGACY_ASSET_DB_PATH` so it is clear what and why this is done